xinference/deploy/cmdline.py

```diff
@@ -640,8 +642,8 @@ def list_model_registrations(
     help='The number of GPUs used by the model, default is "auto".',
 )
 @click.option(
-    "--peft-model-path",
-    default=None,
+    "--peft-model-paths",
+    multiple=True,
     type=str,
     help="PEFT model path.",
 )
```

Comment:
Note that this is cmdline, users cannot pass a List[str] directly. You need to state in the help message that it is comma-separated, and then split into List in the function.